cmd/documentation/communications_templates/telegram.tmpl

```diff
@@ -20,19 +20,22 @@ developed by Telegram Messenger LLP
 	+ Individual package example below:
```

Comment:
Perhaps highlight that the zero placeholder will be replaced by the bot once the authclient is verified. Just because of the difficulty of actually getting the ID otherwise.